test/parallel/test-stream-transform-callback-twice.js

```diff
@@ -7,7 +7,7 @@ const stream = new Transform({
 
 stream.on('error', common.expectsError({
   type: Error,
-  message: 'Callback called multiple times',
+  message: 'The undefined callback was called more than once',
```

Comment:
Referring to the suggestions above, error message on this file is also redefined.
Please suggest me on that!
Thank You! @mhdawson